eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -954,10 +954,10 @@ object Helpers {
       val remoteCommit = d.commitments.remoteCommit
       val nextRemoteCommit_opt = d.commitments.remoteNextCommitInfo.left.toOption.map(_.nextRemoteCommit)
       if (localCommit.publishableTxs.commitTx.tx.txid == tx.txid) {
-        // our commit got confirmed, so any htlc that we signed but they didn't sign will never reach the chain
-        val mostRecentRemoteCommit = nextRemoteCommit_opt.getOrElse(remoteCommit)
+        // our commit got confirmed, so any htlc that is in their commitment but not ours will never reach the chain
```

Comment:
```suggestion
        // our commit got confirmed, so any htlc that is in their commitment but not in ours will never reach the chain
```
maitre_capello.jpg

Reply:
Done in https://github.com/ACINQ/eclair/pull/1706/commits/e2119a400387f34134257c5975f884a23c0c3411